src/views/preview/project-view.jsx

```diff
@@ -246,7 +235,7 @@ class Preview extends React.Component {
     }
     getProjectData (projectId, showAlerts) {
         if (projectId <= 0) return 0;
-        storage
+        return storage
```

Comment:
Is this intentional?

Reply:
No, it's a remnant from the refactor, when this was a promise... removing